code/_globalvars/lists/flavor_misc.dm

```diff
@@ -349,4 +349,4 @@ GLOBAL_LIST_INIT(job_heirlooms, list(
 GLOBAL_LIST_INIT(nongendered_limb_types, list("fly", "zombie" ,"synth", "shadow", "cultgolem", "agent", "plasmaman", "clockgolem", "clothgolem"))
 
 //list of eye types, corresponding to a respective left and right icon state for the set of eyes
-GLOBAL_LIST_INIT(eye_types, list("normal", "insect", "moth", "double", "double2", "double3", "cyclops"))
+GLOBAL_LIST_INIT(eye_types, list("normal", "insect", "moth", "double", "double2", "double3", "cyclops", "beeg"))
```

Comment:
don't make this a meme name

Reply:
this PR is basically dead anyways, the purpose of the b e e g eyes was to allow people to have larger than average eyes without them being covered by hair
but hair still covers them because hairstyles are designed for 1 pixel eyes